addons/website/views/snippets/s_discovery.xml

```diff
@@ -4,15 +4,19 @@
 <template id="s_discovery" name="Discovery">
     <section class="s_discovery pt136 pb136">
         <div class="container" style="text-align: center;">
-            <span class="s_cta_badge o_cc o_cc1 d-inline-block my-3 border rounded py-2 px-3 o_animable" data-snippet="s_cta_badge" data-name="CTA Badge" style="border-radius: 32px !important;">
-                <i class="fa fa-fw fa-info-circle o_not-animable" role="img"/>&#160;&#160;What do you want to promote&amp;nbsp;? &#160;&#160;&#160;&#160;<a href="#">See more <i class="fa fa-long-arrow-right" role="img"/></a>
-            </span>
-            <h1 class="display-2" style="text-align: center;">Discover our solutions</h1>
-            <p class="lead" style="text-align: center;">Write one or two paragraphs describing your product, services or a specific feature.<br/> To be successful your content needs to be useful to your readers.<br/><br/></p>
-            <p style="text-align: center;">
-                <a t-att-href="cta_btn_href" class="btn btn-primary"><t t-out="cta_btn_text">Our store</t></a>
-                <a t-att-href="cta_btn_href" class="btn btn-secondary"><t t-out="cta_btn_text">Contact us</t></a>
-            </p>
+            <div class="row s_nb_column_fixed">
+                <div class="col-12 col-lg-8 offset-lg-2">
```

Comment:
What's the point of this extra row and fixed col if everything is centered anyway? Is that related to this commit's work about buttons?

Reply:
You're right, this diff shouldn't be part of this commit, it's related to [this PR](https://github.com/odoo/odoo/pull/181509), where we need to define a col to prevent the text for taking the whole container size, and even within that PR, it should be widespread to some other snippets, I'll remove it from this commit